src/WPML_OptionsManager.php

```diff
@@ -547,59 +563,55 @@ private function displayMP3Banner() {
             $option = rand(1, 2);
             add_option('wpml_banner_version', $option);
         }
-        ?>
-        <div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;margin-top: 15px;padding: 20px;">
-            <h3><?php _e( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' );?></h3>
-            <?php
-              if ($option === 1) {
-                    $this->displayMP3BannerVersionA();
-              } else {
-                    $this->displayMP3BannerVersionB();
-              }
-            ?>
-        </div>
-        <?php
+
+        $banner = '<div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;margin-top: 15px;padding: 20px;">';
+        $banner .= sprintf('<h3>%s</h3>', __( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' ) );
+
+        if ($option === 1) {
+            $banner .= $this->displayMP3BannerVersionA();
+		} else {
```

Comment:
These changes use tab-based indentation, while the rest of the project uses space-based indentation.
Could you please use space-based indentation as well, matching the coding style of the project?